zetaclient/zetabridge/broadcast.go

```diff
@@ -173,7 +181,13 @@ func (b *ZetaCoreBridge) GetContext() (client.Context, error) {
 	if err != nil {
 		return ctx, err
 	}
-	ctx = ctx.WithClient(wsClient)
+
+	if b.cfg.UseMockSDKClient {
+		ctx = ctx.WithClient(b.mockSDKClient)
+	} else {
+		ctx = ctx.WithClient(wsClient)
```

Comment:
May we can just add `wsClient` as part of the `broadcast` field instead of using `UseMockSDKClient`

Reply:
Can you explain a bit more? Do you mean remove the flag and just use the client that is in the zetabridge struct regardless?

I have thought about different solutions. Most of them are over complicated by the fact that a new context is created for every broadcast. I Think that zetabridge might need some refactoring anyway, there is a config that seems unnecessary  and can just use zetaclient cfg.

Reply:
Ok, let's think about refactoring this part then for the broadcast in a later PR.
I think we can keep it as this now but I would consider removing the `mockSDKClient` in the future as it is